src/app/test_executive/block_reward_test.ml

```diff
@@ -0,0 +1,68 @@
+open Core
+open Integration_test_lib
+open Mina_base
+
+module Make (Inputs : Intf.Test.Inputs_intf) = struct
+  open Inputs
+  open Engine
+  open Dsl
+
+  open Test_common.Make (Inputs)
+
+  (* TODO: find a way to avoid this type alias (first class module signatures restrictions make this tricky) *)
+  type network = Network.t
+
+  type node = Network.Node.t
+
+  type dsl = Dsl.t
+
+  let config =
+    let open Test_config in
+    let open Test_config.Wallet in
+    { default with
+      requires_graphql = true
+    ; block_producers = [ { balance = "1000"; timing = Untimed } ]
+    }
+
+  let run network t =
+    let open Malleable_error.Let_syntax in
+    let logger = Logger.create () in
+    let all_nodes = Network.all_nodes network in
+    let%bind () = wait_for t (Wait_condition.nodes_to_initialize all_nodes) in
+    let[@warning "-8"] [ untimed_node_a ] = Network.block_producers network in
+    let[@warning "-8"] [ bp ] = Network.block_producer_keypairs network in
+    let bp_pk = bp.public_key |> Signature_lib.Public_key.compress in
+    let bp_pk_account_id = Account_id.create bp_pk Token_id.default in
+    let bp_original_balance = Currency.Amount.of_mina_string_exn "1000" in
+    let coinbase_reward = Currency.Amount.of_mina_string_exn "720" in
```

Comment:
Is it necessary to hard-code this value? Couldn't we import it from somewhere

Reply:
No, we can't. Helena is looking into it, but there is a comment elsewhere in the integration tests saying the value can't be imported.